### PR TITLE
Apply Prettier

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Sync existing branches:

1. Check out your local Branch
2. Run `git merge `
3. Resolve any merge conflicts, commit.
4. Run `npm prettier`
5. Commit
6. Run `git merge -Xours `
7. Push

## Testing requirements

N/A Should be no functional changes

## Before you submit
- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by eliykat** [comment]
Taking a look at the first two commits, `b61cfff` and `dbc5b0f`.

**Comment by djsmith85** [comment]
Commits for `src/app/404` and `src/app/accounts` (20665037acd7699a03e4ed2b89df96766daf8753, e95b465b5249621137c743a6c91c3d965b8801e4) are looking good

**Comment by djsmith85** [comment]
Commits for `src/app/common`, `src/app/components`, `src/app/layouts` (a9eb2eaf9423fd1bde4eb7cb29ea7c4761d7e96a and b41111f7b72cd6a9d20a84167b1326529c1542c5) are looking good

**Comment by djsmith85** [comment]
Commits for `src/app/providers`, `src/app/send` (2c12940985e61599808b02bd190ad8aeac6d4176 and 54235eaf2c8ce98db04e9990cf24deba30bfdab6) are looking good

**Comment by djsmith85** [comment]
Commits for `src/app/tools` (0987a1ebdf8748a739b832765f7617355e3ed180 and 60d7c2c448893943e8447b701d1a5fee641ceef8) are looking good

**Comment by djsmith85** [comment]
Commits for `src/app/vault` (3d21de4eb9ebcb42c9382bdf18bda2c33e1fa861 and 1f3d6b7b8ae0c97086295e27a9b80862d799b1ff) are looking good

**Comment by djsmith85** [comment]
Commits for `src/app/services` (8b2989b5936c26291a5909fdd92e541b834a1c17 and c9f332878a41c135dbca825ea7d6906b8f0bfa46) are looking good

**Comment by differsthecat** [comment]
Commits for `Remaining Files` (c717c9b and 762e48a) look good

**Comment by differsthecat** [comment]
Commits for `src/app/organizations` (ea50475 and 803b41b) look good!

**Review by djsmith85 — CHANGES_REQUESTED** [review]
Commits for `src/app/settings` (54196704bcb56dbc8baf8af668bd5c11540d41f7 and 94c73584eb306ed8f9d96c29ac40beeebbd48900) are looking good.

Just one minor thing, with a missing i18n string

File: src/app/settings/organizations.component.ts [hunk 1]
```diff
@@ -1,169 +1,192 @@
...
+
+  async unlinkSso(org: Organization) {
+    const confirmed = await this.platformUtilsService.showDialog(
+      "Are you sure you want to unlink SSO for this organization?",
+      org.name,
+      this.i18nService.t("yes"),
+      this.i18nService.t("no"),
+      "warning"
+    );
+    if (!confirmed) {
```
[the remark this answers is not in the document]
Comment (Hinton) [reply src/app/settings/organizations.component.ts:86]: We'll need to fix it in a separate PR.

[the remark this answers is not in the document]
Comment (djsmith85) [reply src/app/settings/organizations.component.ts:86]: Spoke with @Hinton and this will be fixed in a separate PR

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**